site/content/en/blog/releases/1.27.0-rc.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "1.27.0 - Release Candidate"
+linkTitle: "1.27.0-rc"
+date: "2022-10-18"
+---
+This is the 1.27.0-rc release of Agones.
+
+This release brings us support for Kubernetes 1.23, so if you are installing this release, remember to upgrade your clusters.
+
+Check the README for details on features, installation and usage.
+
+Implemented enhancements:
+
+- Allow cluster autoscaler to scale down game server pods [\#2747](https://github.com/googleforgames/agones/issues/2747)
+- \[GKE\] - Should we enable image streaming everywhere? [\#2746](https://github.com/googleforgames/agones/issues/2746)
+- Support Agones on ARM systems [\#2216](https://github.com/googleforgames/agones/issues/2216)
+- Update example containers to fix security vulnerabilities [\#1154](https://github.com/googleforgames/agones/issues/1154)
+
```

Comment:
Do we also want to callout the [Go version upgrade](https://github.com/googleforgames/agones/pull/2743)

Reply:
That's a good idea. I just added the `kind/feature` label to that PR so it will be included (but we need to re-run the changelog generator to pick it up).